exonum-java-binding-common/src/main/java/com/exonum/binding/common/configuration/StoredConfiguration.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.common.configuration;
+
+import com.exonum.binding.common.hash.HashCode;
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import java.util.List;
+
+/**
+ * Represents a blockchain configuration which is a set of values that determine
+ * the network access parameters of a node and behavior of the node while operating in the network.
+ */
+@AutoValue
+@GenerateTypeAdapter
+public abstract class StoredConfiguration {
```

Comment:
I am not sure what the best source is — it might be exonum.com/doc or the Rust docs of Exonum. @vitvakatu , @MartaDubno , @Maria-Nosyk WDYT?

Anyway, the Javadocs do not have to repeat the docs on the site — but provide consice, accurate and essential information about these properties of the configuration.

Reply:
I don't understand the need of documentation duplication, we already have all docs available online. Would it be enough to provide link for corresponding explanation , with Java details provided additionally?